GPX/GPXTrailsTrackPointExtensions.m

```diff
@@ -0,0 +1,91 @@
+//
+//  GPXTrailsTrackPointExtensions.m
+//  GPX
+//
+//  Created by Jan Weitz on 27.04.2015
+//
+//
+
+#import "GPXTrailsTrackPointExtensions.h"
+#import "GPXElementSubclass.h"
+
+NSString *const kElementHorizontalAcc = @"trailsio:hacc";
+NSString *const kElementVerticalAcc = @"trailsio:vacc";
+NSString *const kElementSteps = @"trailsio:steps";
+NSString *const kTagName = @"trailsio:TrackPointExtension";
+
+@interface GPXTrailsTrackPointExtensions ()
+
+@property (nonatomic) NSString *horizontalAccuracyString;
+@property (nonatomic) NSString *verticalAccuracyString;
+@property (nonatomic) NSString *stepCountString;
+
+@end
+
+@implementation GPXTrailsTrackPointExtensions
+
+- (id)initWithXMLElement:(GPXXMLElement *)element parent:(GPXElement *)parent {
+    self = [super initWithXMLElement:element parent:parent];
+    
+    if (self) {
+        _horizontalAccuracyString = [self textForSingleChildElementNamed:kElementHorizontalAcc xmlElement:element];
+        _verticalAccuracyString = [self textForSingleChildElementNamed:kElementVerticalAcc xmlElement:element];
+        _stepCountString = [self textForSingleChildElementNamed:kElementSteps xmlElement:element];
+    }
+    
+    return self;
+}
+
+#pragma mark - Public methods
+- (void)setHorizontalAccuracy:(NSNumber *)horizontalAccuracy {
```

Comment:
would we want to distinguish between setting `@(0)` and `nil`? This way `nil` becomes @"0" becomes `@(0)` on the way back in `- horizontalAccuracy`. Same for the other properties.